event_sale_reservation/migrations/15.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """UPDATE product_product SET detailed_type = 'event_reservation'
+        WHERE event_reservation_ok IS TRUE""",
```

Comment:
Boolean, so no need to do an extra equal operation

```suggestion
        WHERE event_reservation_ok""",
```